www/app/Chat.tsx

```diff
@@ -291,7 +274,8 @@ export default function Chat({
         metadata: {},
       },
     ];
-    mutateMessages(newMessages, { revalidate: false });
+    await mutateMessages(newMessages, { revalidate: false });
```

Comment:
Is the await necessary I noticed you didn't use it in all locations

Reply:
await only needs to be used if there's a guaranteed immediate change in the window height, so it makes sure to update scroll state after